sr_demos/scripts/sr_demos/demo.py

```diff
@@ -491,6 +527,7 @@ def grasp_demo(self):
         self.commander.move_to_named_target("open")
 
         rospy.loginfo("Grasp Demo completed")
+        self._enable_touch_detection(True)
 
     def complete_random_sequence(self):
```

Comment:
No type hinting?

Reply:
fixed in https://github.com/shadow-robot/sr_interface/pull/823/commits/4d899740e14c6a37b20146403ea06d3b82a837c8